DAOS-15937 test: DO NOT LAND TEST ONLY

kccain opened the pull request:
Run daos_test -c, co_op_dup_timing test on several CI clusters to establish baseline timing for pools with svc_ops_enabled:1 and svc_ops_entry_age:60.

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_daos_container
Skip-func-test: true
Skip-func-hw-test-medium-verbs-provider: true
Skip-func-hw-test-medium-verbs-provider-md-on-ssd: true
Skip-func-hw-test-medium-ucx-provider: true
Skip-func-hw-test-medium: false
Skip-func-hw-test-medium-md-on-ssd: false

Test-repeat: 5

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

kccain committed:
DAOS-15937 test: DO NOT LAND TEST ONLY

Run daos_test -c, co_op_dup_timing test on several CI
clusters to establish baseline timing for pools with
svc_ops_enabled:1 and svc_ops_entry_age:60.

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_daos_container
Skip-func-hw-test-medium-verbs-provider: false
Skip-func-hw-test-medium-verbs-provider-md-on-ssd: false
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium: false
Skip-func-hw-test-medium-md-on-ssd: false

Test-repeat: 5

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

kccain committed:
remove provider tag to run on func hw medium and medium mdssd

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_daos_container
Skip-func-hw-test-medium-verbs-provider: true
Skip-func-hw-test-medium-verbs-provider-md-on-ssd: true
Skip-func-hw-test-medium-ucx-provider: true
Skip-func-hw-test-medium: false
Skip-func-hw-test-medium-md-on-ssd: false

Test-repeat: 5

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14997/2/display/redirect

kccain committed:
add a 1000 iteration container open/close loop after warmup

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_daos_container
Skip-func-test-vm: true
Skip-func-hw-test-medium-verbs-provider: true
Skip-func-hw-test-medium-verbs-provider-md-on-ssd: true
Skip-func-hw-test-medium-ucx-provider: true
Skip-func-hw-test-medium: false
Skip-func-hw-test-medium-md-on-ssd: false

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

daosbuild1 commented:
Test stage Functional on EL 8.8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-14997/5/execution/node/811/log
